yb-voyager/cmd/live_migration_integration_test.go

```diff
@@ -0,0 +1,711 @@
+
+	reportFilePath := filepath.Join(exportDir, "reports", "data-migration-report.json")
+	// Check if the report file exists and is not empty.
+	if ok := utils.FileOrFolderExists(reportFilePath); ok {
+		jsonFile := jsonfile.NewJsonFile[[]*rowData](reportFilePath)
+		rowData, err := jsonFile.Read()
+		testutils.FatalIfError(t, err, "error reading get data-migration-report")
+		exportSnapshot := 0
+		importSnapshot := 0
+		for _, row := range *rowData {
+			if row.TableName == tableName {
+				if row.DBType == "source" {
+					exportSnapshot = int(row.ExportedSnapshotRows)
+				}
+				if row.DBType == "target" {
+					importSnapshot = int(row.ImportedSnapshotRows)
+				}
+			}
+		}
+		if exportSnapshot == int(snapshotRows) && exportSnapshot == importSnapshot {
+			return true
+		}
+
+	}
+	return false
+}
+
+// checks for the streaming phase completed in export and import both by checking the get data migration report
+func streamingPhaseCompleted(t *testing.T, postgresPass string, targetPass string, streamingInserts int64, tableName string) bool {
+	err := testutils.NewVoyagerCommandRunner(nil, "get data-migration-report", []string{
+		"--export-dir", exportDir,
+		"--output-format", "json",
+		"--source-db-password", postgresPass,
+		"--target-db-password", targetPass,
+	}, nil, true).Run()
+	testutils.FatalIfError(t, err, "get data-migration-report command failed")
+
+	reportFilePath := filepath.Join(exportDir, "reports", "data-migration-report.json")
+	// Check if the report file exists and is not empty.
+	if ok := utils.FileOrFolderExists(reportFilePath); ok {
+		jsonFile := jsonfile.NewJsonFile[[]*rowData](reportFilePath)
+		rowData, err := jsonFile.Read()
+		testutils.FatalIfError(t, err, "error reading get data-migration-report")
+		exportInserts := 0
+		importInserts := 0
+		for _, row := range *rowData {
+			if row.TableName == tableName {
+				if row.DBType == "source" {
+					exportInserts = int(row.ExportedInserts)
+				}
+				if row.DBType == "target" {
+					importInserts = int(row.ImportedInserts)
+				}
+
+			}
+		}
+		if exportInserts == int(streamingInserts) && exportInserts == importInserts {
+			return true
+		}
+
+	}
+	return false
+}
+
+// This inserts some rows in target table having sequence and validates if the ids ingested are correct or not
+func assertSequenceValues(t *testing.T, startID int, endId int, ybConn *sql.DB, tableName string) {
+	_, err := ybConn.Exec(fmt.Sprintf(`INSERT INTO test_schema.test_live (name, email, description)
+SELECT
+	md5(random()::text),                                      -- name
+	md5(random()::text) || '@example.com',                    -- email
+	repeat(md5(random()::text), 10)                           -- description (~320 chars)
+FROM generate_series(%d, %d);`, startID, endId))
+	testutils.FatalIfError(t, err, "inserting into target")
+
+	ids := []string{}
+	for i := startID; i <= endId; i++ {
+		ids = append(ids, strconv.Itoa(i))
```

Comment:
everything can be kept integer? i think no need of string

Reply:
yeah but I putting it in SELECT query as well so modified it to changed only there rest all can still be int